calligraphy/src/main/java/uk/co/chrisjenx/calligraphy3/CalligraphyFactory.java

```diff
@@ -15,6 +15,8 @@
 import java.lang.ref.WeakReference;
 import java.lang.reflect.Method;
 
+import io.github.inflationx.viewpump.ReflectionUtils;
+
 class CalligraphyFactory {
```

Comment:
Do you think this should still be called `Factory`? The `ViewPump#Interceptor` calls into these methods right?